pyccel/ast/basic.py

```diff
@@ -9,11 +9,15 @@
 - Basic, which provides a python AST
 - PyccelAstNode which describes each PyccelAstNode
 """
-
+import ast
 from sympy.core.basic import Basic as sp_Basic
 
 __all__ = ('Basic', 'PyccelAstNode')
 
+dict_keys   = {}.keys().__class__
+dict_values = {}.values().__class__
```

Comment:
Is this any different than writing

```python
dict_keys   = type({}.keys())
dict_values = type({}.values())
```
?

Reply:
No, it is the same. I can use this syntax if you think it is clearer

Reply:
Fixed in 9a1a43ee

Reply:
Yes please, I think that using `type()` is more natural.

Reply:
Fixed in 9a1a43e

Reply:
I cannot find the changes..

Reply:
Hmm, I don't know where they went. Fixed now : 590c687

Reply:
Thanks